app/template.rb

```diff
@@ -2,9 +2,6 @@
 copy_file "app/controllers/concerns/basic_auth.rb"
 copy_file "app/helpers/layout_helper.rb"
 
-remove_dir "app/jobs"
-empty_directory_with_keep_file "app/workers"
```

Comment:
No need to go against the grain. Just use ActiveJob, which is built into Rails. Sidekiq works with it nicely.